gap/examples.gi

```diff
@@ -370,3 +370,43 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(GeneralisedPetersenGraphCons,
+"for IsImmutableDigraph, integer, int",
+[IsImmutableDigraph, IsInt, IsInt],
+function(filt, n, k)
+  local D;
+  D := MakeImmutable(GeneralisedPetersenGraphCons(IsMutableDigraph, n, k));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  return D;
+end);
```

Comment:
Ah ha! @LRacine I have worked out why the code coverage test was failing. It's because you have accidentally duplicated this method for `GeneralisedPetersenGraphCons`. Since only one of the methods is ever run, that means that one of the copies has absolutely no coverage. So if you delete it the coverage test should pass 🙂 
```suggestion
```